### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by PososikTeam**

### CODE REVIEW

**Review by VladVin — CHANGES_REQUESTED** [review]

File: torchok/__main__.py [hunk 1]
```diff
@@ -30,10 +31,21 @@ def entrypoint(config: DictConfig):
...
+        seed_everything(**config.seed_params)
     # Create task
     model = torchok.TASKS.get(config.task.name)(config)
     trainer = create_trainer(config)
     if entrypoint == 'train':
+        if config.trainer.auto_lr_find:
+            # Run learning rate finder
+            lr_finder = trainer.tuner.lr_find(model)
+            suggested_lr = lr_finder.suggestion()
+            for i in range(len(config.optimization)):
```
Comment (Animatory) [inline torchok/__main__.py:45]: PL does not support several optimizers. Throw error if got this more than one optimizers

File: torchok/tasks/base.py [hunk 2]
```diff
@@ -118,15 +119,13 @@ def __check_drop_last_params(self, data_params: List[Dict[str, Any]], phase: str
                 raise ValueError(f'DataLoader parameters `drop_last` must be False in {phase} phase.')
 
     def on_train_start(self) -> None:
-        if self.current_epoch == 0:
-            load_checkpoint(self, base_ckpt_path=self._hparams.task.base_checkpoint,
-                            overridden_name2ckpt_path=self._hparams.task.overridden_checkpoints,
-                            exclude_keys=self._hparams.task.exclude_keys)
+        if self.current_epoch == 0 and self._hparams.load_checkpoint is not None:
+            load_checkpoint(self, **self._hparams.load_checkpoint)
+        optim = self.optimizers()
```
Comment (Animatory) [inline torchok/tasks/base.py:124]: Remove

File: torchok/constructor/config_structure.py [hunk 3]
```diff
@@ -154,6 +154,13 @@ class TrainerParams:
     move_metrics_to_cpu: bool = False
     multiple_trainloader_mode: str = "max_size_cycle"
 
+# Load checkpoint params
+@dataclass
+class LoadCheckpointParams:
```
Comment (Animatory) [inline torchok/constructor/config_structure.py:159]: Move in TaskParams

File: torchok/__main__.py [hunk 4]
```diff
@@ -30,10 +31,21 @@ def entrypoint(config: DictConfig):
...
+                config.optimization[i].optimizer.params.lr = suggested_lr
```
[the remark this answers is not in the document]
Comment (PososikTeam) [reply torchok/__main__.py:46]: Yes it's a common way. https://pytorch-lightning.readthedocs.io/en/1.4.5/advanced/lr_finder.html#:~:text=model%20%3D%20MyModelClass(hparams,trainer.fit(model)

**Review by Animatory — APPROVED** [review]

**Review by VladVin — APPROVED** [review]

**Pull request merged**